### ISSUE TITLE
Create Punjabi language queries

### ISSUE THREAD

**Issue by SethiShreya** [scribe-org/Scribe-Data#271]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would look into expanding the [src/scribe_data/language_data_extraction/Punjabi](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/language_data_extraction/Punjabi) files with as much data as are possible from the current data on Wikidata. We can use code for getting data from other languages, and from there we can check Punjabi data on Wikidata for what conjugations are available. We can then expand the query with optional selections of certain forms as is done in other SPARQL queries. The query can be tried on the [Wikidata Query Service UI](https://query.wikidata.org/) during development :)

Data types to include:
- [x] Nouns
- [x] Verbs
- [x] Adjectives
- [x] Adverbs
- [x] Prepositions
- [x] Emoji keywords

### Contribution

I will start with creating nouns, gender and verb queries for Punjabi and then will take it further to add conjugations and other stuffs.

**Comment by andrewtavis** [comment]
Thanks for the issue, @SethiShreya! :)

**Comment by SethiShreya** [comment]
Added Sparql query for Punjabi language nouns, so when I was searching I found while extracting the nouns for punjabi language, it was extracting for Gurmukhi (Punjabi language for India) and Shahmukhi (Punjabi language for Muslims), So I added filter only to get Gurmukhi with tag ("pa"). 
The query also included the Gender, plural and singular nouns for Gurmukhi. I have tested it on website and in the code too.

Please review and give your feedbacks

**Comment by andrewtavis** [comment]
Hey @SethiShreya 👋 Interesting to learn all of this. Recently we've been discussing moving some languages to a common sub directory so it's a bit more organized, so would you say that we could have a `Punjabi` directory and have `Gurmukhi` and `Shahmukhi` sub directories? The same can be seen now in Norwegian :) Let me know. I'll review the current PR as is :)

**Comment by SethiShreya** [comment]
> Hey @SethiShreya 👋 Interesting to learn all of this. Recently we've been discussing moving some languages to a common sub directory so it's a bit more organized, so would you say that we could have a `Punjabi` directory and have `Gurmukhi` and `Shahmukhi` sub directories? The same can be seen now in Norwegian :) Let me know. I'll review the current PR as is :)

That would be good, so directory structure will be

Punjabi
     Gurmukhi
     Shahmukhi

Right?

**Comment by andrewtavis** [comment]
Yes, exactly. We'll do this maybe in a week as the new directories can't be run anyway :) Feel free to change the structure in your next PR!

**Comment by andrewtavis** [comment]
Just added a list of data types that we want to include to this issue :) We can maybe do all of this for Gurmukhi and Shahmukhi, @SethiShreya 😊

**Comment by SethiShreya** [comment]
Thanks @andrewtavis, sure i will work on those data types and on the directory structure too.

**Comment by andrewtavis** [comment]
Sounds great, @SethiShreya! :)

**Comment by SethiShreya** [comment]
@andrewtavis i have bifurcated directory to Gurmukhi and Shahmukhi added verb query and fixed directory structure to call these too. Raised a PR [here](https://github.com/scribe-org/Scribe-Data/pull/331), please check once. Checked it on my end.

please review and give your feedback, Thanks :)

**Comment by andrewtavis** [comment]
Sent along the base versions of the rest of the queries in f14a335, @SethiShreya :) Let me know if you'd like to expand them with their forms :)

**Comment by GicharuElvis** [comment]
Hi, anyone working on this?

**Comment by andrewtavis** [comment]
Hey @GicharuElvis 👋 Appreciate your interest in the project :) We'll be using new functionality to auto-generate these queries soon, so it will be closed in the coming days. Please let us know if there's anything else you find interesting to work on!

**Comment by GicharuElvis** [comment]
sure, if its possible i'd also like to help in the automation.

**Comment by andrewtavis** [comment]
Would be great, @GicharuElvis! Automation will likely be happening over at [Scribe-Server](https://github.com/scribe-org/Scribe-Server), which will be running Scribe-Data :) Do you have skills/interest in Go? We could maybe find an issue over there :)

**Comment by andrewtavis** [comment]
Closing this as work here will continue in #513 where we'll be autogenerating the queries :) Thanks all for the work here, and hope all are well! 😊
